canvas.md

```diff
@@ -1,17 +1,58 @@
 # Research Canvas
 
-Attention is all you need (Vaswani, et. al. 2017)
-
 ## Key Literature
 
-* Long short term memory (Hochreiter, Schmidhuber 1997)
-* .. .more references
+1. Y. Zheng, Z. Wang, Z. Huang, T. Jiang, "Comovement between the Chinese Business Cycle and Financial Volatility: Based on a DCC-MIDAS Model", *Emerging Markets Finance and Trade*, Vol. 56, No. 6, 2020.

```

Comment:
This is a long list.

A future step for you will be to complete a literature review. In that process you may find that not all these papers are relevant, or you may find that there are some currently unlisted papers that are very important. Please keep this in mind.